tests/integration/targets/dhcp_option_group/tasks/main.yml

```
@@ -0,0 +1,230 @@
+
+    - name: Create a DHCP option Group with tags
+      infoblox.universal_ddi.dhcp_option_group:
+          name: "{{ option_group_name }}"
+          tags:
+              location: "my-location"
+          state: "present"
+      register: option_group
+    - name: Get information about the DHCP Option Group
+      infoblox.universal_ddi.dhcp_option_group_info:
+          filters:
+              name: "{{ option_group_name }}"
+      register: option_group_info
+    - assert:
+        that:
+          - option_group is changed
+          - option_group is not failed
+          - option_group_info is not failed
+          - option_group_info.objects | length == 1
+          - option_group_info.objects[0].id == option_group.id
+          - option_group_info.objects[0].name == option_group_name
+          - option_group_info.objects[0].tags.location == "my-location"
+
+    - name: Create a second DHCP option Group
+      infoblox.universal_ddi.dhcp_option_group:
+          name: "{{ option_group_name }}_2"
+          state: "present"
+      register: option_group_2
+    - name: Get information about the DHCP Option Group
+      infoblox.universal_ddi.dhcp_option_group_info:
+          filters:
+              name: "{{ option_group_name }}_2"
+      register: option_group_info_2
+    - assert:
+        that:
+          - option_group_2 is changed
+          - option_group_2 is not failed
+          - option_group_info_2 is not failed
+          - option_group_info_2.objects | length == 1
+          - option_group_info_2.objects[0].id == option_group_2.id
+          - option_group_info_2.objects[0].name == option_group_2.object.name
+
+    - name: Create a DHCP Option for the DHCP Option Group
+      infoblox.universal_ddi.dhcp_option_group:
+          name: "{{ option_group_name }}"
+          dhcp_options:
+            - type: "group"
+              group: "{{ option_group_2.id }}"
+            - type: "option"
+              option_code: "{{_option_code.id}}"
+              option_value: "False"
+          state: "present"
+      register: option_group
+    - name: Get information about the DHCP Option
+      infoblox.universal_ddi.dhcp_option_group_info:
+          filters:
+              name: "{{ option_group_name }}"
+      register: option_group_info
+    - assert:
+        that:
+          - option_group is changed
+          - option_group is not failed
+          - option_group_info is not failed
+          - option_group_info.objects | length == 1
+          - option_group_info.objects[0].id == option_group.id
+          - option_group_info.objects[0].name == option_group_name
+          - option_group_info.objects[0].dhcp_options[0].type == "group"
+          - option_group_info.objects[0].dhcp_options[0].group == option_group_2.id
+
+  always:
+    - name: Delete the DHCP Option Group
+      infoblox.universal_ddi.dhcp_option_group:
+          name: "{{ option_group_name }}"
+          state: "absent"
+      ignore_errors: yes
```

```suggestion
      ignore_errors: true
```

Please change all occurrences of ignore_errors